workbench-core/datasets-ui/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@aws/workbench-core-datasets-ui",
+  "version": "0.0.1",
+  "description": "Package that contains UI components for Datasets",
+  "homepage": "https://github.com/awslabs/solution-spark-on-aws",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/awslabs/solution-spark-on-aws.git"
+  },
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Amazon Web Services",
+    "url": "http://aws.amazon.com"
+  },
+  "main": "lib/esm/index.js",
+  "module": "./lib/esm/index.js",
+  "typings": "lib/esm/index.d.ts",
+  "scripts": {
+    "add-license-header": "license-check-and-add add -f license-add-config.json",
+    "build": "heft build --clean && rushx pkg-json-lint",
+    "build:test": "heft test --clean && rushx pkg-json-lint && rushx make-badges",
+    "check-license-header": "license-check-and-add check -f license-add-config.json",
+    "depcheck": "depcheck",
+    "license-checker": "license-checker --onlyAllow 'MIT; Apache-2.0; ISC; BSD'",
+    "lint": "eslint . && rushx pkg-json-lint",
+    "lint:fix": "eslint . --fix",
+    "make-badges": "istanbul-badges-readme --coverageDir=./temp/coverage --exitCode=1",
+    "pkg-json-lint": "npmPkgJsonLint -c ../../.npmpackagejsonlintrc.json .",
+    "sort-package-json": "sort-package-json package.json",
+    "test": "rushx test:only && rushx make-badges",
+    "test:only": "heft test --clean --no-build"
+  },
+  "dependencies": {
```

Comment:
have you run `rushx depcheck`? It doesnt look like all these dependencies are needed